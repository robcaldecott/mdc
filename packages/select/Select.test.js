import React from "react";
import { Select } from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(
    renderer.create(
      <Select value="" label="Pick a food group">
        <option value="grains">Bread, Cereal, Rice, and Pasta</option>
        <option value="vegetables" disabled>
          Vegetables
        </option>
        <option value="fruit">Fruit</option>
        <option value="dairy">Milk, Yogurt, and Cheese</option>
        <option value="meat">
          Meat, Poultry, Fish, Dry Beans, Eggs, and Nuts
        </option>
        <option value="fats">Fats, Oils, and Sweets</option>
      </Select>
    )
  ).toMatchSnapshot();
});
