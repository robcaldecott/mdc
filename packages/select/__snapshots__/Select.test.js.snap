// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="mdc-select mdc-select--disabled"
>
  <select
    className="mdc-select__surface"
    disabled={true}
    value=""
  >
    <option
      value=""
    >
      Pick a food group
    </option>
    <option
      value="grains"
    >
      Bread, Cereal, Rice, and Pasta
    </option>
    <option
      disabled={true}
      value="vegetables"
    >
      Vegetables
    </option>
    <option
      value="fruit"
    >
      Fruit
    </option>
    <option
      value="dairy"
    >
      Milk, Yogurt, and Cheese
    </option>
    <option
      value="meat"
    >
      Meat, Poultry, Fish, Dry Beans, Eggs, and Nuts
    </option>
    <option
      value="fats"
    >
      Fats, Oils, and Sweets
    </option>
  </select>
  <div
    className="mdc-select__bottom-line"
  />
</div>
`;
